api-participant/src/main/java/bio/terra/pearl/api/participant/controller/EventController.java

```diff
@@ -0,0 +1,31 @@
+package bio.terra.pearl.api.participant.controller;
+
+import bio.terra.pearl.api.participant.api.EventApi;
+import bio.terra.pearl.core.service.events.MixpanelService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class EventController implements EventApi {
```

Comment:
let's consolidate this with LoggingController.  Also, let's look into whether we should drop/refactor our existing loggingUtils.tsx and use mixpanel to instrument JS errors and web vitals (I think we should)